src/Stratis.Bitcoin.IntegrationTests/BlockStore/ReorgToLongestChainSteps.cs

```diff
@@ -22,9 +22,10 @@ public partial class ReorgToLongestChainSpecification
         private CoreNode daveNode;
         private Transaction shorterChainTransaction;
         private int jingsBlockHeight;
+
         private const string AccountZero = "account 0";
-        private const string WalletZero = "wallet 0";
-        private const string WalletPassword = "123456";
+        private const string WalletZero = "mywallet";
```

Comment:
Is this still needed?

Reply:
Yeah it is... although `WalletPassphrase ` is not ;)